Preprocessing/LabelEncoder.ecl

```diff
@@ -0,0 +1,201 @@
+/*###################################################################################
+## HPCC SYSTEMS software Copyright (C) 2020 - 2021 HPCC Systems.  All rights reserved.
+##################################################################################### */
+
+/**
+ * Allows to convert categorical features' values into numbers
+ * in the range [0 - n] where n is the number of categories for a feature.
+ *
+ * Note: currently not support Myraid interface.
```

Comment:
Should read: "Currently does not support Myriad interface".  Myriad also has a typo.